lms/static/sass/_build-lms-v2.scss

```diff
@@ -3,7 +3,22 @@
 // Version 2 - introduces the Pattern Library
 
 
-// Configuration
```

Comment:
Yeah, my nickname in high school was "ugly separators". :P Glad you guys are giving this so much love.